lib/active_encode/engine_adapters/test_adapter.rb

```diff
@@ -9,23 +9,28 @@ def create(encode)
         new_encode = encode.dup
         new_encode.id = SecureRandom.uuid
         new_encode.state = :running
+        new_encode.created_at = Time.now
+        new_encode.updated_at = Time.now
         @encodes[new_encode.id] = new_encode
         new_encode
       end
 
       def find(id, _opts = {})
-        @encodes[id].dup
+        encode = @encodes[id]
+        # Update the updated_at time to simulate changes
+        encode.updated_at = Time.now
+        encode
       end
 
       def list(*_filters)
         raise NotImplementedError
       end
 
```

Comment:
previously find always return a new copy of the encode, now we return the onsite instance. is this change due to that we are not consider this as rails engine instead of gem?

Reply:
Good catch!  I forgot about that explicit change when merging master into this branch.  I'll fix that now.